code/game/machinery/deployable.dm

```diff
@@ -132,6 +132,14 @@
 	AddElement(/datum/element/climbable)
 	AddElement(/datum/element/elevation, pixel_shift = 12)
 
+/obj/structure/barricade/sandbags/atom_deconstruct(disassembled = TRUE)
+	var/atom/Tsec = drop_location()
```

Comment:
this name is real bad. what is it exactly? is it a contraction if so just put the whole thing

Reply:
It seemed like the standard that other many other structures used for this, so I went with it. Anything better that I should change it to?

Reply:
well like, what is the var? is it the spawn location? is it the place we're dropping stuff?

Reply:
In truth you may not like this answer: this is how it's done in so many other places in the code, I figured I would just follow along and do the same. If I should do something else I will.

Reply:
please give it a name that describes what it actually does here